server/controller/User.controller.js

```diff
@@ -0,0 +1,97 @@
+const User = require('../models/Users.model');
+
+const getAllUser = async (req, res) => {
+  try {
+    const users = await User.find({});
+    return res.status(200).json({ users, sucess: true });
+  } catch (error) {
+    console.log(error);
+    return res.status(500).json(error);
+  }
+};
+
+const createUser = async (req, res) => {
+  try {
+    const {
+      name, email, age, password, cpassword,
+    } = req.body;
+    if (!name) {
```

Comment:
Alot of if statements. make a single if with or validation and the response should be Invalid or missing input data for now. Later we will use middleware for this validation.